CLOUD/Get-AzureSizingInfo.ps1

```diff
@@ -1128,6 +1131,31 @@ Write-Host "Output files are:"
 $outputFiles.Files
 Write-Host
 
+$archiveFile = "azure_results_$($date.ToString('yyyy-MM-dd_HHmm')).zip"
+
+Write-Host
+Write-Host "Results will be compressed into $archiveFile and original files will be removed." -ForegroundColor Green
+
+Stop-Transcript
+
+$outputFiles += New-Object -TypeName PSCustomObject -Property @{Files="output.log - Log file"}
+
+# Extract only the unique file names from the array of objects for compression
+$filePaths = $outputFiles | ForEach-Object { $_.Files.Split(' - ')[0] }  | Sort-Object -Unique
+
+# Compress the files into a zip archive
+Compress-Archive -Path $filePaths -DestinationPath $archiveFile
+
+# Remove the original files
+foreach ($file in $filePaths) {
+    Remove-Item -Path $file -ErrorAction SilentlyContinue
+}
+
+Write-Host
+Write-Host
+Write-Host "Results have been compressed into $archiveFile and original files have been removed." -ForegroundColor Green
```

Comment:
@vpritamani-rubrik, please add an instructional line after this, telling the user to send the zip file to their Rubrik representative.